assets/js/translator.js

```diff
@@ -1,5 +1,6 @@
 // @flow
-
+var languages;
+var iso639CodesInverse;
```

Comment:
These should be `/*:: import`d from the correct modules, not re-declared like this,